src/client/index.d.ts

```diff
@@ -68,3 +68,7 @@ declare module "@vx/tooltip" {
   }
   const TooltipWithBounds: React.ComponentType<TooltipWithBoundsProps>;
 }
+
+declare module 'highlight.js/lib/languages/json' {
```

Comment:
`node_modules/highlight.js/types/index.d.ts` has something like this. Not sure why but TS doesn't seem to see it.

Reply:
I don't have f... idea. I tried typeRoots, installing all different combination of packages and nothing :o

For completeness I would add `highlight.js` to dependencies and change definition slightly.

```
declare module "highlight.js/lib/languages/json" {
  import { LanguageFn } from "highlight.js";
  const defineLanguage: LanguageFn;
  export default defineLanguage;
}
```

Reply:
Weird, I've just noticed that without this module type declaration locally type checking pass but it fails on CI 🤔 @adrianmroz-allegro Could you confirm please? 🙏

Reply:
I got type error locally without this declaration module. I would keep this inside `client.d.ts` for completeness because typescript clearly can't get its hands on definitions in highlight/types.